Cloud/src/main/java/com/example/petfeedercloud/services/PetFeederServiceImpl.java

```diff
@@ -179,17 +188,11 @@ private PetFeeder convertToEntity(PetFeederDTO petFeederDTO) {
         petFeeder.setFoodLevel(petFeederDTO.getFoodLevel());
         petFeeder.setFoodHumidity(petFeederDTO.getFoodHumidity());
         petFeeder.setWaterTemperture(petFeederDTO.getWaterTemperture());
+        petFeeder.setActive(false);
+        petFeeder.setConnected(false);
```

Comment:
when deactivating a pet feeder only active is set to false. Once a pet feeder is connected there is now way to unconnect it. If they dont want the petfeeder the delete it.